### ISSUE TITLE
Add publication date to insights

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#780]
A new **optional** field should be added to /insights/add with name `Publication Date`.

**Comment by SimonLab** [comment]
Just to clarify the publication date is the date when the author of the insight created the document (or other type of insights) or is it the date when a user of the application create the insight itself? I guess the first one?

**Comment by asimonw** [comment]
@SimonLab No, it's the date when the document was written by the author. This is important to indicate relevance. But nothing to worry about now. I created the issue to make sure we add it in a later sprint.
